hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/pipeline/Pipeline.java

```diff
@@ -443,7 +443,7 @@ public static Builder newBuilder(Pipeline pipeline) {
     return new Builder(pipeline);
   }
 
-  private void setReplicaIndexes(Map<DatanodeDetails, Integer> replicaIndexes) {
+  public void setReplicaIndexes(Map<DatanodeDetails, Integer> replicaIndexes) {
```

Comment:
I think this should remain private to keep Pipeline immutable.

Reply:
I agree. I think I missed as it was used later after build. I will correct that usage as well.